src/StagedProposalProcessor.sol

```diff
@@ -637,6 +613,35 @@ contract StagedProposalProcessor is ProposalUpgradeable, PluginUUPSUpgradeable {
         }
     }
 
+    /// @notice Internal function to advance the proposal.
+    /// @dev Note that is assumes the proposal can advance.
+    /// @param _proposalId The proposal Id.
+    function _advanceProposal(uint256 _proposalId) internal {
```

Comment:
can you make this `virtual` as well just in case ?

Reply:
done in https://github.com/aragon/staged-proposal-processor-plugin/pull/13/commits/2246f5ab2b774e99ab5c1384c1c7a1925edfdd4e